plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/BaseDeltaLakeConnectorSmokeTest.java

```diff
@@ -2249,6 +2255,55 @@ public void testPartitionFilterIncluded()
         }
     }
 
+    @Test(dataProviderClass = DataProviders.class, dataProvider = "trueFalse", invocationCount = 3)
```

Comment:
BaseConnectorTest already has testInsertRowConcurrently. Why not copy the test to BaseConnectorSmokeTest?

Reply:
```
            assertQuery("SELECT version, operation, read_version FROM \"" + tableName + "$history\"",
                    """
                            VALUES
                                (0, 'CREATE TABLE', 0),
                                (1, 'WRITE', 0),
                                (2, 'WRITE', 1),
                                (3, 'WRITE', 2)
                            """);
```

The `$history` table check is Delta Lake dependent.